metabase/metabase
# Fix updating query fields in viz settings

Changes:
- Migrate available column selection in viz settings to MLv2
- Migrate syncing column viz settings of a structured query to MLv2
- Convert related components to TS

How to verify:
- Go to viz settings and toggle columns off and on
- Make sure that "fields" clause is changed in the query accordingly
- Try it with plain tables, aggregated data, joins, questions based on other questions etc
